packages/moonstone/Scrollable/Scrollbar.module.less

```diff
@@ -64,5 +65,10 @@
 	/* ScrollThumb */
 	.thumbShown {
 		opacity: 1;
+		.applySkins({

```

Comment:
Our convention has been to use `.applySkins()` under the root class and duplicate the hierarchy within it. Should we do the same here?